src/components/taskList/taskList.css

```diff
@@ -3,10 +3,13 @@
   font-weight: var(--fw-primary-light);
   display: flex;
```

Comment:
Ich würde flex rausnehmen. Label Element ist ein Block Element, deswegen werden Elemente auch ohne Flex runter untereinander brechen

Reply:
Good idea, thank you!